src/utils/explorer-api.js

```diff
@@ -12,22 +12,16 @@ export async function getTransactions(accountId = '') {
     const tx = await new Promise((resolve, reject) => wamp.call(
         `${WAMP_NEAR_EXPLORER_TOPIC_PREFIX}.select`,
         [
-            `
-                SELECT 
-                    transactions.hash,
-                    transactions.signer_id, 
-                    transactions.receiver_id, 
-                    transactions.actions, 
-                    transactions.block_hash, 
-                    blocks.timestamp as blockTimestamp
-                FROM 
-                    transactions
-                LEFT JOIN blocks ON blocks.hash = transactions.block_hash
-                WHERE 
-                    signer_id = :accountId 
-                    OR receiver_id = :accountId
-                ORDER BY blocks.height DESC
-                LIMIT :offset, :count
+            `SELECT 
+                    hash, signer_id, receiver_id, block_hash, 
+                    block_timestamp, action_type as kind, action_args as args
+            from 
+                (SELECT hash, signer_id,  receiver_id, block_hash,  block_timestamp
+                    FROM  transactions
+                    WHERE  signer_id = :accountId OR receiver_id = :accountId
+                    ORDER BY block_timestamp DESC
+                    LIMIT :offset ,:count) as transactions
+                    LEFT JOIN actions ON actions.transaction_hash = transactions.hash
```

Comment:
nit: needs to be pulled back in indentation IMO as now it looks like this join is part of inner `(SELECT`, but in fact it is part of outer `SELECT`.